scenario/BulkerScenario.ts

```diff
@@ -38,7 +38,7 @@ scenario(
     expect(await comet.balanceOf(albert.address)).to.be.equal(0n);
 
     // Albert's actions:
-    // 1. Supplies 100 units of collateral
+    // 1. Supplies 1000 units of collateral
```

Comment:
This test currently assumes that 100 units of `asset0` are worth 1500 of the base unit.

The price of LINK on Goerli is currently around $7.

Bumping up the supplied collateral to cover the borrow,